angular-portfolio/.editorconfig

```diff
@@ -0,0 +1,16 @@
+# Editor configuration, see https://editorconfig.org
```

Comment:
This file can probably be added to the .gitignore file. This will mean it will stay on your computer but not be uploaded to git. You should also probably run "git rm --cached FILENAME" as described in this file: https://help.github.com/en/github/using-git/ignoring-files